todos-app/src/App1.tsx

```diff
@@ -0,0 +1,46 @@
+import { KeyboardEvent, useState } from "react";
+import reactLogo from "./assets/react.svg";
+import "./App.css";
+("use strict");
+
+function RenderApp() {
+  const keyup = (event: KeyboardEvent): void => {
+    if (event.key !== "Enter") {
+      return;
+    }
+
+    event.preventDefault();
+
+    if (event.key === "Enter") {
+      const inputMessage = getInput("new-input");
+      console.log(inputMessage, inputMessage.length);
+
+      //console.log(getInput("new-input"));
```

Comment:
Seems like there are 3 files that have App*.tsx... ideally, there should only be one App.tsx

Reply:
Fixed it. I just forgot to delete these files before uploading to GitHub